### ISSUE TITLE
Timeline progresses non-smoothly

### ISSUE THREAD

**Issue by MDBijman** [atlarge-research/opendc-frontend#9]
The timeline of a simulation is expected to progress smoothly, i.e. the time the browser waits between each step should be the same. Instead what happens is that the timeline moves 2-3 seconds very quickly, and then waits a little while, repeating until the end.

**Comment by gandreadis** [comment]
@MDBijman: I understand why that may seem unintuitve, let me explain my reasoning behind it. The timeline has transitions between ticks, and stops on each tick. I am aware that this may not be how users are accustomed to visual playback timelines, since they might expect them to actually move linearly/continuously. However, in those kinds of (commonly video) players, the content does have more frames between different ticks. 

Our simulations, however, don't interpolate states between ticks (something for the future 😉), which is why I felt it unfitting to let it flow smoothly through intermediate steps. Let me know if you feel differently about this - I have no problem changing this (shouldn't be too hard in terms of code).

(BTW: I don't think those are 2-3 seconds, I'm quite certain that they move in 1s between ticks)

**Comment by MDBijman** [comment]
@gandreadis: I think there's some misunderstanding in what I meant. I agree with you that stepping one tick, waiting, stepping one tick, etc. is fine. The problem is that this is not what was happening when I created the issue. It would quickly step over several ticks, then wait a bit longer, etc. I just tried it again and it seems to work, but I'm not on the same system so I can't confirm that it has been fixed.
